modules/exoplanets/exoplanetsmodule_lua.inl

```diff
@@ -236,7 +245,7 @@ void createExoplanetSystem(const std::string& starName) {
 
     // Planets
     for (size_t i = 0; i < system.planetNames.size(); i++) {
-        ExoplanetDataEntry planet = system.planetsData[i];
+        ExoplanetDataEntry& planet = system.planetsData[i];
```

Comment:
seems like this could be `const ExoplanetDataEntry&`?

Reply:
It can not, since some member values of `planet` are replaced with default values later in the code if they are nan. 

This will be improved after my data reading rewrite, after which I hope to not have to handle nan values explicitly in the scene graph node creation code anymore :)